specifications/api/swagger.yaml

```diff
@@ -798,16 +798,21 @@ paths:
       tags: ["Wallets"]
       summary: List
       description: |
-        Priority [Very High]
+        <p align="right">status: <strong>incomplete¹</strong></p>
```

Comment:
Interesting that it is possible to embed HTML within these fields. Does this embedding of HTML affect interoperability with other tools that visualize swagger specifications?

Reply:
Hmmm. 

1. No.
2. No -- shouldn't.

The swagger specification is quite clear on that and says:

```
A short description of the application. GFM syntax can be used for rich text representation.
```

Where `GFM` here stands for [Github Flavored Markdown](https://guides.github.com/features/mastering-markdown/#GitHub-flavored-markdown). Being markdown means that, we are allowed to put basic HTML too since that's what markdown is the end :)

Reply:
_It means we should be able to add emojis too_

**O**-**M**-**G**.

Reply:
Understood. Thanks for the clarification!